frontend/src/scenes/insights/LineGraph/LineGraph.tsx

```diff
@@ -361,7 +361,7 @@ function LineGraphV2(props: LineGraphProps): JSX.Element {
                                         hideColorCol={isHorizontal || !!tooltipConfig?.hideColorCol}
                                         renderCount={tooltipConfig?.renderCount}
                                         forceEntitiesAsColumns={isHorizontal}
-                                        hideInspectActorsSection={!(onClick && showPersonsModal)}
+                                        hideInspectActorsSection={!onClick || !showPersonsModal}
```

Comment:
Hmm, sorry for not noticing before, but seeing this perhaps we could unify `showPersonsModal` with `onClick`? As in, it doesn't seem like `showPersonsModal={true}` can work with `onClick={undefined}`, while `onClick={() => ...)` should not be set with `showPersonsModal={false}` as that'd be needlessly hiding the feature from the user.

Reply:
Well, I suppose in that case we'd just pass `onClick` as `undefined` (in `InsightCard` for instance), but it's not a big deal anyway